plugin/collector/snap-plugin-collector-mock2/mock/mock.go

```diff
@@ -81,6 +81,15 @@ func (f *Mock) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, err
 			panic("Oops!")
 		}
 
+		if c, ok := mts[i].Config().Table()["time_to_wait"]; ok {
+			dur, err := time.ParseDuration(c.(ctypes.ConfigValueStr).Value)
+			if err != nil {
+				log.Println(err.Error())
+			} else {
+				time.Sleep(dur)
+			}
+		}
+
```

Comment:
@rashmigottipati, I know that introducing `time_to_wait` in snap-plugin-collector-mock2 was really helpful for testing gracefully shutdown behavior during working on it. However, I have a doubt about landing this change on master - is it really needed? Does it provide some value - I mean is there any large test which runs mock2 collector and testing this behavior? If yes, that is fine - I won't insist on removing that.
 -